backend/inspirehep/records/api/base.py

```diff
@@ -387,6 +406,7 @@ def update(self, data, *args, **kwargs):
             if data.get("deleted"):
                 self.pidstore_handler.delete(self.id, self)
             else:
+                self.delete_records_from_record_deleted(data)
```

Comment:
what about create? 🤔

Reply:
Ah, you are right :+1:  . It's OK for literature (as it's calling update inside create) but not for other types.